moto/ec2/models.py

```diff
@@ -5562,6 +5562,30 @@ def modify_spot_fleet_request(
         return True
 
 
+class SpotPrice(object):
```

Comment:
This should be named `SpotPriceBackend`, to make it consistent with other backends